graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/ReloadSchemaIndicator.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Netflix, Inc.
+ * Copyright 2025 Netflix, Inc.
```

Comment:
why change it here? there are 50 other places with 2024 and hundreds with <=2023, I thought it's based on file introduction year

Reply:
oh actually that makes sense. I relocated this file to the core module, I think that's where it belongs. But that probably made it a "new" file